frappe/commands/site.py

```diff
@@ -1275,8 +1297,52 @@ def handle_data(data: dict, format="json"):
 		render_table(data)
 
 
+def add_new_user(
+		email, first_name=None, last_name=None, send_welcome_email=False, password=None,role=None):
+	# add user
+	user = frappe.new_doc("User")
+	user.update(
+		{
+			"name": email,
+			"email": email,
+			"enabled": 1,
+			"first_name": first_name or email,
+			"last_name": last_name,
+			"user_type": "System User",
+			"send_welcome_email": 1 if send_welcome_email else 0,
+		}
+	)
+	user.insert()
+	user.add_roles(*role)
+	if password:
+		from frappe.utils.password import update_password
+		update_password(user=user.name, pwd=password)
+
+
+@click.command("list-roles")
```

Comment:
Hey,

You can use the `execute` command (ref: https://frappeframework.com/docs/v13/user/en/bench/frappe-commands) for this. Eg:

`bench --site {name} execute frappe.get_roles`
`bench --site {name} execute frappe.get_list --args 'Role' --kwargs '{"pluck":"name"}'`

Either commands will give you a machine readable output to STDOUT

Reply:
OK perfect we will remove the list-roles command